backend/doc/user_custom_properties.md

```diff
@@ -0,0 +1,98 @@
+# User Custom Properties
+User custom properties is a feature allowing to add instance-specific properties
+for all user in the instance.
+
+Idea is that for each user is associated some properties
+which structure is defined by a JSONSchema draftv7 (specification available [here]('https://json-schema.org/specification-links.html#draft-7'))
+
+Another specific schema, more related to the way things should be display is available too. it's
+named UISchema and base specification are available [here]('https://react-jsonschema-form.readthedocs.io/en/latest/api-reference/uiSchema/').
+
+We do support also some tracim specific properties to have some tracim specifics behaviors in both schema.
+We provide you also a way to translate theses schema.
```

Comment:
```suggestion
User-visible parts of these properties (title/description/…) can be translated ...
```
We are missing a description of the translation process of the user properties here.

Reply:
is need documentation ok ?